docs/documentation.html

```diff
@@ -746,8 +746,16 @@ <h3 id="importing-external-rules">Importing External Rules</h3>
        grammar, NOT the source.  Grammars MUST be compiled by a version that
        supports imports in order to be imported.  Only rules that are allowed
        start rules are valid.  It can be useful to specify
-       <code>--allowed-start-rules \*</code> in library grammars. All of the
-       following are valid:</p>
+       <code>--allowed-start-rules \*</code> in library grammars.  Imports are
+       only valid in output formats "es" and "commonjs".  If you
+       use imports, you should use <code>{ output: "source" }</code>; the default
+       output of "parser" will call `eval` on the source which fails immediately
+       for some formats (e.g. "es") and will not find modules in the expected
+       places for others (e.g. "commonjs").  The
+       ["from-mem"](https://github.com/peggyjs/from-mem/) project is used by
+       the Peggy CLI to resolve these issues, but note well its relatively severe
+       limitations.</p>
+       <p>All of the following are valid:</p>
```

Comment:
Something is missing here or am i wrong?

Reply:
I've fixed the format issue for the link.  What else do you see?